torch_tvm/fusion_pass.cpp

```diff
@@ -59,10 +59,26 @@ c10::optional<Node*> tryMerge(
       consumer->kind().toQualString(),
       ":\n");
 
-  // Symbolic checks
-  REQ(canHandle(producer, aliasDb));
+  // Symbolic check
   REQ((canHandle(consumer, aliasDb) || consumer->kind() == getTVMSymbol()));
 
+  // if producer cannot be converted, check if consumer can be lowered to TVM
```

Comment:
I think we should do this outside of this function. `tryMerge` is supposed to merge two nodes into a lowerable node. Iideally we start with a seed outside. Maybe 
```
if (canLowerSeed(consumer
```
If seed cannnot be lowered, we never come to `tryMerge`?

Also @bwasti, maybe you should chime in here.

Reply:
Yeah, I think we should put this kind of logic in parallel to tryMerge instead of inside. I actually think we should have some kind of flag to control minimal number of ops in order to create a fusion group. But this might not be the scope of the PR.

Reply:
I initially implemented as two passes:
1) Lower all nodes that can be lowered
2) Do Fusion among nodes that are already lowered

1 works fine but fusing two tvm nodes does not quire work as is.